src/App.jsx

```diff
@@ -65,9 +93,15 @@ function App() {
           <Navbar />
           <Routes>
             <Route index element={<Home />} />
+            <Route path="sell" element={<Sell />} />
+            <Route path="buy" element={<Buy />} />
+            <Route path="rent" element={<Rent />} />
             <Route path="cardpage" element={<CardPage />} />
             <Route path="about" element={<About />} />
             <Route path="blog" element={<Blog />} />
+            <Route path="blog/grodno" element={<Grodno />} />
```

Comment:
Could be nested inside Parent route component, simply add the same <Routes>/<Route> structure to the top of the <Blog /> component